src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CollectionBackedStoreProvider.cs

```diff
@@ -47,5 +48,10 @@ public void Remove(ICertificatePal cert)
         {
             throw new InvalidOperationException();
         }
+
+        SafeHandle IStorePal.SafeHandle
```

Comment:
I tried to hide it a bit as there was already an internal SafeCertStoreHandle property that I didn't want to replace because it is preferred over SafeHandle for the main implementation code to see\use because of type safety and because Unix doesn't know about SafeCertStoreHandle .